microsoft/runbook/tiers/t1.yml

```diff
@@ -2,7 +2,17 @@ variable:
   - name: v2_rg
     value: v2-t1
 testcase:
+  # v3 p0, p1 test cases
   - criteria:
       priority: [0, 1]
+  # LisaV2 P0 functional, excluding XDP, DPDK, LIS_DEPLOY
   - type: legacy
-    command: .\Run-LisaV2.ps1 -TestLocation '$(location)' -RGIdentifier '$(v2_rg)' -TestPlatform 'Azure' -ARMImageName '$(image)' -XMLSecretFile '$(v2_secret_file)' -TestPriority 0 -ResourceCleanup $(v2_rg_cleanup)
+    repo: $(v2_repo_url)
+    branch: $(v2_branch)
+    command: .\Run-LisaV2.ps1 -TestLocation '$(location)' -RGIdentifier '$(v2_rg)' -TestPlatform 'Azure' -EnableTelemetry -ARMImageName '$(gallery_image)' -XMLSecretFile '$(v2_secret_file)' -TestPriority 0 -ResourceCleanup '$(v2_rg_cleanup)' -CustomLIS '$(custom_lis)' -CustomTestParameters '$(custom_test_parameters)' -OverrideVMSize '$(override_vmsize)' -CustomKernel '$(custom_kernel)' -ExcludeTests '*DPDK*,*XDP*,^LIS-DEPLOY*,^LIS-DRIVER*' -ForceCustom -RunInParallel -TotalCountInParallel 3
```

Comment:
Wait, do we have commands coming from runbooks? And hard-coded to ps1 scripts? Confused.

Reply:
It's a good chance to validate below document. Could you try to find answers from it? Let me know, if any question here.
https://github.com/microsoft/lisa/blob/main/docs/run_legacy.md

Reply:
I see, so we edit things on these runbooks at the pace tests are migrated/translated, then.

Reply:
yes, we're simplifying and creating test standard.